draft-bryce-cose-merkle-mountain-range-proofs.md

```diff
@@ -60,9 +61,17 @@ The storage of a tree maintained in this way is addressed as a linear array, and
 In this specification, all numbers are unsigned 64 bit integers.
 The maximum height of a single tree is 64 (which will have `g=63` for its peak).
 
-# Verifiable Data Structure
+# Description of the MMR Ledger Verifiable Data Structure
 
-The integer identifier for this Verifiable Data Structure is 2.
+This documents extends the verifiable data structure registry of {{-COMTRE}} with the following value:
+
+| Name | Value | Description | Reference
+|---
+|MMRivr_LEDGER_SHA256 | TBD_1 (requested assignment 3) | Merkle Mountain Range Ledgers, such as the MMRivr ledger | This document
+{: #verifiable-data-structure-values align="left" title="Verifiable Data Structure Algorithms"}
+
+This document defines inclusion proofs for Merkle Mountain Range (MMRivr) ledgers.
```

Comment:
MMRIVER or MMR or something else like CCF is confidential compute consortia. MMRivr isn't anything

Reply:
"MMR" has the benefit of being shorter in these exploded type_and_alg names.
"MMRIVER" has the benefit of being specific to this kind of MMR implementation (as distinct from the many other ways of implementing and exploiting MMRs).

It's very important to accept the requested assignment 3 but I suggest we leave the name alone

Reply:
Due to the subtle differences, and advantages, Robin and I went with MMRIVER for the reasons above.